:man_farmer: Build regression CI Windows JointPositionController

<!-- If you're not sure whether your problem is a bug, please ask a question at
http://answers.gazebosim.org instead.-->

## Environment
* OS Version: Windows
* Source
## Description

* Expected behavior: CI builds `ign-gazebo` properly
* Actual behavior: It's failing.

## Steps to reproduce
Attempt to build Windows from source.

Reference CI Job: https://build.osrfoundation.org/job/ign_gazebo-ci-win/134/

## Output
<!-- Provide screenshots, console logs, backtraces, and/or anything that could
be useful to us resolving this issue -->

This section of the error is red:
```
`ign-gazebo\src\systems\joint_position_controller\JointPositionController.cc(74,5): error C2143: syntax error: missing '}' before 'constant'`
```
I think this one was introduced with this PR: https://github.com/ignitionrobotics/ign-gazebo/pull/1003. 

Probably the defined enum variable defined `ABSOLUTE` is also a windows compiler keyword. See: https://stackoverflow.com/questions/8139738/visual-studio-compiler-errors-at-enum

--- a/src/systems/joint_position_controller/JointPositionController.cc
+++ b/src/systems/joint_position_controller/JointPositionController.cc
@@ -66,12 +66,13 @@ class ignition::gazebo::systems::JointPositionControllerPrivate
   public: unsigned int jointIndex = 0u;
 
   /// \brief Operation modes
-  enum OperationMode {
+  enum OperationMode
+  {
     /// \brief Use PID to achieve positional control
     PID,
     /// \brief Bypass PID completely. This means the joint will move to that
     /// position bypassing the physics engine.
-    ABSOLUTE
+    ABS
   };
 
   /// \brief Joint position mode
@@ -162,7 +163,7 @@ void JointPositionController::Configure(const Entity &_entity,
     if (useVelocityCommands)
     {
       this->dataPtr->mode =
-        JointPositionControllerPrivate::OperationMode::ABSOLUTE;
+        JointPositionControllerPrivate::OperationMode::ABS;
     }
   }
 
@@ -270,9 +271,9 @@ void JointPositionController::PreUpdate(
             this->dataPtr->jointPosCmd;
   }
 
-  // Check if the mode is ABSOLUTE
+  // Check if the mode is ABS
   if (this->dataPtr->mode ==
-    JointPositionControllerPrivate::OperationMode::ABSOLUTE)
+    JointPositionControllerPrivate::OperationMode::ABS)
   {
     // Calculate target velcity
     double targetVel = 0;

